t/mojo/bytestream.t

```diff
@@ -77,49 +78,54 @@ ok !b('hell')->secure_compare('hello'), 'values are not equal';
 is b('hello')->xor_encode('foo'),                "\x0e\x0a\x03\x0a\x00", 'right result';
 is b("\x0e\x0a\x03\x0a\x00")->xor_encode('foo'), 'hello',                'right result';
 
-# Nested bytestreams
-my $stream = b(b('test'));
-ok !ref $stream->to_string, 'nested bytestream stringified';
-$stream = Mojo::ByteStream->new(Mojo::ByteStream->new('test'));
-ok !ref $stream->to_string, 'nested bytestream stringified';
-
-# split
-$stream = b('1,2,3,4,5');
-is_deeply $stream->split(',')->to_array,   [1, 2, 3, 4, 5], 'right elements';
-is_deeply $stream->split(qr/,/)->to_array, [1, 2, 3, 4, 5], 'right elements';
-is_deeply b('1,2,3,4,5,,,')->split(',')->to_array, [1, 2, 3, 4, 5], 'right elements';
-is_deeply b('1,2,3,4,5,,,')->split(',', -1)->to_array, [1, 2, 3, 4, 5, '', '', ''], 'right elements';
-is_deeply b('54321')->split('')->to_array, [5, 4, 3, 2, 1], 'right elements';
-is_deeply b('')->split('')->to_array,    [], 'no elements';
-is_deeply b('')->split(',')->to_array,   [], 'no elements';
-is_deeply b('')->split(qr/,/)->to_array, [], 'no elements';
-$stream = b('1/2/3');
-is $stream->split('/')->map(sub { $_->quote })->join(', '), '"1", "2", "3"', 'right result';
-is $stream->split('/')->map(sub { shift->quote })->join(', '), '"1", "2", "3"', 'right result';
+subtest 'Nested bytestreams' => sub {
+  my $stream = b(b('test'));
+  ok !ref $stream->to_string, 'nested bytestream stringified';
+  $stream = Mojo::ByteStream->new(Mojo::ByteStream->new('test'));
+  ok !ref $stream->to_string, 'nested bytestream stringified';
+};
+
+subtest 'split' => sub {
+  my $stream = b('1,2,3,4,5');
+  is_deeply $stream->split(',')->to_array,   [1, 2, 3, 4, 5], 'right elements';
+  is_deeply $stream->split(qr/,/)->to_array, [1, 2, 3, 4, 5], 'right elements';
+  is_deeply b('1,2,3,4,5,,,')->split(',')->to_array, [1, 2, 3, 4, 5], 'right elements';
+  is_deeply b('1,2,3,4,5,,,')->split(',', -1)->to_array, [1, 2, 3, 4, 5, '', '', ''], 'right elements';
+  is_deeply b('54321')->split('')->to_array, [5, 4, 3, 2, 1], 'right elements';
+  is_deeply b('')->split('')->to_array,    [], 'no elements';
+  is_deeply b('')->split(',')->to_array,   [], 'no elements';
+  is_deeply b('')->split(qr/,/)->to_array, [], 'no elements';
+  $stream = b('1/2/3');
+  is $stream->split('/')->map(sub { $_->quote })->join(', '), '"1", "2", "3"', 'right result';
+  is $stream->split('/')->map(sub { shift->quote })->join(', '), '"1", "2", "3"', 'right result';
+};
```

Comment:
Why is this test only partially converted?

Reply:
All converted now.